pkgs/applications/version-management/smartgithg/default.nix

```diff
@@ -34,7 +34,11 @@ stdenv.mkDerivation rec {
       --prefix JRE_HOME : ${jre} \
       --prefix JAVA_HOME : ${jre} \
       --prefix SMARTGITHG_JAVA_HOME : ${jre} \
-    ) \
+    )
+ 
+    # add missing shebang for start script
+    sed -i $out/bin/smartgit \
+      -e '1i#!/bin/bash'
```

Comment:
`/bin/bash` does not exist on nixos.
You can use `${pkgs.bash}/bin/bash` instead.

Reply:
/bin/bash will be replaced in post-installation fixup as far as i know and i has testet